tests/main.tftest.hcl

```
@@ -213,6 +213,33 @@ run "test_default_example_stack_final_values" {
   }
 }
 
+# Test the default-example stack with only 1 runtime override
+run "test_default_example_stack_partial_runtime_overrides" {
+  command = plan
+
+  variables {
+    runtime_overrides = {
+      root-module-a = {
+        stack_settings = {
+          administrator = false
```

@westonplatter you just got the name here wrong. Try this: 
```suggestion
          administrative = false
```

---

ah, that would make a difference. Updated that in the commits below.